src/Raven.Studio/typescript/models/database/cluster/clusterTopology.ts

```diff
@@ -60,10 +61,26 @@ class clusterTopology {
             }
         });
 
+        this.updateAssignedCores(incomingChanges.AssignedCoresByNode);
         this.nodeTag(incomingChanges.NodeTag);
         this.leader(incomingChanges.Leader);
         this.currentTerm(incomingChanges.CurrentTerm);
     }
+
+    private updateAssignedCores(assignedCoresByNode: { [index: string]: number; }) {
+        for (let nodeTag in assignedCoresByNode) {
```

Comment:
this can be simplied using:
```
_.forOwn(assignedCoresByNode, (cores, nodeTag) {
   const node = this.nodes().find(x => x.tag() === nodeTag);
  if (node) {
    node.assignedCores(cores);
  }
});
```

Reply:
done